.github/ISSUE_TEMPLATE/deploy-request.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Deploy Request
+about: Request deploy prow
+title: 'REQUEST: deploy prow for <your-GH-handle>'
+labels: type/deploy
+assignees: 'hi-rustin'
+---
+
+### GitHub Repo Name
+e.g. (at)example_repo
+
+### Which plugins do you want to deploy
+You can find all available plugins in [this manual book](https://book.prow.tidb.io/).
```

Comment:
I think we can make a table about all plugins in the introduction section of the book. Because some plugin functions are relatively simple and do not appear in the documentation, they can be listed on the table.

插件名称 | 插件类型 | 监听的事件 | 插件功能
-- | -- | -- | --
tide | 基础组件 |   |  
rerere | 基础组件 |   |  
ti-community-owners | 外部插件 |   |  
ti-community-lgtm | 外部插件 |   |  
ti-community-merge | 外部插件 |   |  
ti-community-blunderbuss | 外部插件 |   |  
ti-community-autoresponer | 外部插件 |   |  
ti-community-tars | 外部插件 |   |  
ti-community-label | 外部插件 |   |  
hold | 内置插件 |   |  
assign | 内置插件 |   |  
size | 内置插件 |   |  
lifecycle | 内置插件 |   |  
wip | 内置插件 |   |  
welcome | 内置插件 |   |  
require-matching-label | 内置插件 |   |  
label_sync | 工具 |   |  

Reply:
Cool

Reply:
Create a issue to follow it. https://github.com/ti-community-infra/ti-community-prow/issues/243

Could you please merge it now?

Reply:
OK